website/addons/wiki/views.py

```diff
@@ -143,23 +143,23 @@ def wiki_widget(**kwargs):
     # Show "Read more" link if there are multiple pages or has > 400 characters
     more = len(node.wiki_pages_current.keys()) >= 2
     MAX_DISPLAY_LENGTH = 400
-    use_python_render = False
+    rendered_before_update = False
     if wiki_page and wiki_page.html(node):
         wiki_html = wiki_page.html(node)
         if len(wiki_html) > MAX_DISPLAY_LENGTH:
             wiki_html = BeautifulSoup(wiki_html[:MAX_DISPLAY_LENGTH] + '...', 'html.parser')
             more = True
         else:
             wiki_html = BeautifulSoup(wiki_html)
-        use_python_render = wiki_page.rendered_before_update
+            rendered_before_update = wiki_page.rendered_before_update
```

Comment:
Should this be in the `else` (or `len(wiki_html) <= MAX_DISPLAY_LENGTH`) block? Seems like it should be dedented to be always happen if there's wiki content, and perhaps put above this conditional for clarity.